src/style.css

```
@@ -0,0 +1,31 @@
+:root {
+  --star-size: 16px;
+}
+
+.main-content {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+/*  s*/
+.stars__item {
+  width: var(--star-size);
+  height: var(--star-size);
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__item:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__item:nth-child(-n + 1),
+.stars--2 .stars__item:nth-child(-n + 2),
+.stars--3 .stars__item:nth-child(-n + 3),
+.stars--4 .stars__item:nth-child(-n + 4),
+.stars--5 .stars__item:nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat center;
```

Please note that the only value you want to change here is a background image.
```suggestion
  background-image: url(./images/star-active.svg);
```